### PR TITLE
Amish/4136 i18n

### PR DESCRIPTION
**Pull request by Schabaani**

### CODE REVIEW

**Comment by desfero** [comment]
Closing for now

File: packages/wallet/package.json [hunk 1]
```diff
@@ -13,6 +13,10 @@
     "pods:install:fix": "yarn pods:install --fix",
     "format": "prettier --check '**/*.{ts,tsx,js,json,md,gql}'",
     "format:fix": "prettier --write '**/*.{ts,tsx,js,json,md,gql}' --loglevel warn",
+    "formatjs extract": "prettier --check '**/*.{ts,tsx,js,json,md,gql}'",
```
Comment (desfero) [inline packages/wallet/package.json:16]: some copy paste?

File: packages/wallet/package.json [hunk 2]
```diff
@@ -13,6 +13,10 @@
...
+    "intl:extract": "yarn intl:extract:formatjs && node ./scripts/merge-messages-with-locales.js",
+    "intl:extract:check": "yarn intl:extract:formatjs && node ./scripts/merge-messages-with-locales.js --check",
+    "intl:extract:formatjs": "formatjs extract 'app/**/*.tsx' --out-file app/intl/tmp/en-en.json",
```
Comment (desfero) [inline packages/wallet/package.json:19]: ok, i'm wondering if we can move whole process to a custom made script?
We can use Node API https://formatjs.io/docs/tooling/cli#node-api to call extract messages.

Also would be good to set https://formatjs.io/docs/tooling/cli#--throws flag to get an error if something goes wrong

File: packages/wallet/scripts/merge-messages-with-locales.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
+const { join } = require("path");
+const { readFileSync, writeFileSync } = require("fs");
+const difference = require("lodash/difference");
+
+const defaultLocale = "en-en";
+
+function readFileOrDefault(path, def) {
+  try {
+    const fileContents = readFileSync(path, "utf8") || def;
```
Comment (desfero) [inline packages/wallet/scripts/merge-messages-with-locales.js:9]: i'm wondering why do we need `|| def` given it should throw when file doesn't exist and then we push default value?

File: packages/wallet/app/intl/README.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# Intl
```
Comment (desfero) [inline packages/wallet/app/intl/README.md:1]: 🎖️

File: packages/wallet/app/intl/en-en.json [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+  "modal.send-tx-signer.tx-name.sign-investment-agreement": "Investment Agreement",
+  "modal.send-tx-signer.tx-name.transfer-tokens": "Transfer",
+  "modal.send-tx-signer.tx-name.unlock-funds": "Unlock Funds",
+  "modal.send-tx-signer.tx-name.upgrade": "ICBM Upgrade",
+  "modal.send-tx-signer.tx-name.user-claim": "Claim",
+  "modal.send-tx-signer.tx-name.withdraw": "Withdraw",
+  "modal.sign-message-signer.title": "Sign message request",
+  "modal.wallet-connect-session-request-signer.body": "{peerName} is requesting to connect to your account via {link}",
+  "modal.wallet-connect-session-request-signer.title": "Connection request",
+  "screen.account-backup.write-down-phrase-warning": "If you lose access to your device, the only way to restore your funds and account is with your recovery phrase or private key.\n\nPlease write down and store it in a safe place.",
```
Comment (desfero) [inline packages/wallet/app/intl/en-en.json:49]: Can you make sure `\n\n` works properly?

**Review by desfero — CHANGES_REQUESTED** [review]
Small changes

File: packages/wallet/app/intl/en-en.json (hunk 5 above)
Comment (arjunu) [reply packages/wallet/app/intl/en-en.json:49]: works

File: packages/wallet/package.json (hunk 1 above)
Comment (arjunu) [reply packages/wallet/package.json:16]: oops removed

File: packages/wallet/scripts/merge-messages-with-locales.js (hunk 3 above)
Comment (arjunu) [reply packages/wallet/scripts/merge-messages-with-locales.js:9]: good point, removed

File: packages/wallet/package.json (hunk 2 above)
Comment (arjunu) [reply packages/wallet/package.json]: updated

File: packages/wallet/app/intl/en-en.json (hunk 5 above)
Comment (desfero) [reply packages/wallet/app/intl/en-en.json:49]: cool

**Review by desfero — APPROVED** [review]

**Pull request merged**